src/current/v25.1/critical-log-messages.md

```diff
@@ -198,7 +198,7 @@ toc: true
     - **Action**: Check for details of metrics in the message.
     - **Related metrics**: 
         - `requests.slow.latch`: Number of requests that have been stuck for a long time acquiring latches. Latches moderate access to the KV keyspace for the purpose of evaluating and replicating commands. A slow latch acquisition attempt is often caused by another request holding and not releasing its latches in a timely manner. This in turn can either be caused by a long delay in evaluation (for example, under severe system overload) or by delays at the replication layer. This gauge registering a nonzero value usually indicates a serious problem and should be investigated.
-        - `requests.slow.raft`: Number of requests that have been stuck for a long time in the replication layer. An (evaluated) request has to pass through the replication layer, notably the quota pool and raft. If it fails to do so within a highly permissive duration, the gauge is incremented (and decremented again once the request is either applied or returns an error).
```

Comment:
@kvoli i'm deleting the mention of quota pool b/c IIRC it was mentioned in the eng design doc that quota pool is obsoleted by RACv2, please correct me tho

Reply:
That is correct, we can stop referencing it. It is still around in the case where the user rolls back to `apply_to_elastic`, however no need to confuse users by still mentioning it imo.

Reply:
sweet, i love deleting stuff, will leave it deleted - thanks :-)